kytos/core/auth.py

```diff
@@ -44,6 +50,83 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class UserController:
+    """UserController"""
+
+    # pylint: disable=unnecessary-lambda
+    def __init__(self, get_mongo=lambda: Mongo()):
+        self.mongo = get_mongo()
+        self.db_client = self.mongo.client
+        self.db = self.db_client[self.mongo.db_name]
+
+    def bootstrap_indexes(self):
+        """Bootstrap all topology related indexes."""
+        index_tuples = [
+            ("users", [("username", pymongo.ASCENDING)]),
+        ]
+        for collection, keys in index_tuples:
+            if self.mongo.bootstrap_index(collection, keys):
+                LOG.info(
+                    f"Created DB index {keys}, collection: {collection}"
+                )
+
+    def create_user(self, user_data):
```

Comment:
Let's add type hints on these methods since they're new and relatively simple for this case, it facilitates for other folks to quickly peek at the definition and understand what type the method takes and returns.

Reply:
Added.